source/dynamical_systems/include/dynamical_systems/Circular.hpp

```diff
@@ -110,7 +117,7 @@ class Circular : public DynamicalSystem<state_representation::CartesianState> {
   void set_normal_gain(double gain);
 
   /**
-   * @brief Getter of the radiuses of the limit cycle
+   * @brief Getter of the radius's of the limit cycle
    * @return the radius value
    */
 
```

Comment:
The docstrings for `Getter of the rotation angle attribute` and `Getter of the radiuses of the limit cycle` are swapped around for some reason (plus they each have an extra whitespace). Would be nice to fix while we are here.

Reply:
Where is the extra whitespace?